atomsci/ddm/utils/struct_utils.py

```diff
@@ -288,7 +286,7 @@ def base_mol_from_inchi(inchi_str, useIsomericSmiles=True, removeCharges=False):
         str: Standardized salt-stripped SMILES string.
 
     """
-    if type(inchi_str) != str:
+    if not isinstance(inchi_str, str):
```

Comment:
test?

Reply:
These tests should cover it. https://github.com/ATOMScience-org/AMPL/commit/8246be5a91e0e4999343198956c5085573903396